### PR TITLE
Bring all Dockerfiles for supported Python runtimes into FDK repo

### PR DESCRIPTION
**Pull request by denismakogon**
Bring all Dockerfiles for supported Python runtimes into FDK repo.

### CODE REVIEW

File: images/README.md [hunk 1]
```diff
@@ -3,12 +3,22 @@ Image on Docker Hub: https://hub.docker.com/r/fnproject/python
 ## Building Python images
 
 ```sh
-pushd build/3.7.1; docker build -t fnproject/python:3.7.1-dev .; popd
```
Comment (rdallman) [inline images/README.md]: this file should just be a script probably, but this is ok for now

**Comment by rdallman** [comment]
can you remove from fnproject/dockers too please?

File: images/README.md (hunk 1 above)
Comment (denismakogon) [reply images/README.md]: that's something in my TODO, there's a need to do automated releases as well that is not the case for now.

**Comment by denismakogon** [comment]
> can you remove from fnproject/dockers too please?

done, see https://github.com/fnproject/dockers/pull/23

File: images/README.md (hunk 1 above)
Comment (rdallman) [reply images/README.md]: https://github.com/fnproject/dockers/blob/master/python/build.rb

**Pull request merged**